src/Renderer/Camera.js

```diff
@@ -124,11 +124,13 @@ Camera.prototype.computeNodeSSE = function(node) {
     var boundingSphere = node.geometry.boundingSphere;
     var distance = Math.max(0.0, (this.camera3D.position.distanceTo(node.centerSphere) - boundingSphere.radius));
     // Added small oblique weight (distance is not enough, tile orientation is needed)
+    /*
```

Comment:
and I add  link  in [ main comment PR #194](https://github.com/iTowns/itowns2/pull/194#issue-181982034)